src/lib/openjp2/j2k.c

```diff
@@ -10442,6 +10442,9 @@ opj_codestream_info_v2_t* j2k_get_cstr_info(opj_j2k_t* p_j2k)
     if (!cstr_info) {
         return NULL;
     }
+	
+	cstr_info->nbasoc = 0;
```

Comment:
Indentation issue. Please build with cmake -DWITH_ASTYLE=ON and run scripts/prepare-commit.sh

Reply:
I just did that, but with no effect. Is the prepare-commit.sh Windows-compatible?

Reply:
Hum probably not... Maybe under Cygwin ? Anyway on already committed code, this has no effect. You can use scripts/astyle.sh your.c to fix an existing committed file

Reply:
I did not find an exact match of astyle in Code::Blocks. The closest I got was Kernighan&Ritchie but had to make small edits in source files manually. I guess the 
build system checks if this will pass?
![astyle_kr](https://user-images.githubusercontent.com/37577776/37781347-cf52da5a-2df0-11e8-96b2-862e29e91391.png)

Reply:
Latest push clearly changed everything. My attempt with astyle failed completely.

Reply:
I am afraid I need your assistance to format the committed six files since it can not be done on Windows. I compiled on a remote Linux machine WITH_ASTYLE, but it does not have C++11 and I am not authorized to install gcc. Can you please run astyle.sh on the commited files?

Reply:
OK, I've run astyle. Please pull from rouault:associated_data_support to get https://github.com/rouault/openjpeg/commit/713e13157eeacdf2805a760064165fab04992c87